0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -155,7 +154,8 @@ The mapping from the ILP address to the local ledger address is dependent on the
 #### getBalance
 <code>ledgerPlugin.getBalance() ⇒ Promise.&lt;String></code>
 
-Return an integer string representing the current balance. Plugin must be connected, otherwise the promise should reject.
+Return an integer string representing the current balance, in the ledger's base unit. For example, if the ledger's `currencyCode` is 'EUR' and its `currencyScale` is 5, then
```

Comment:
We should clarify what "base unit" means, exactly, e.g. on a `USD` ledger with a scale of 6, the base unit would be micro-dollars.